Release/src/http/client/http_client_asio.cpp

```diff
@@ -1459,14 +1462,11 @@ class asio_context : public request_context, public std::enable_shared_from_this
                 return true;
             }
 
-            if (!verify_cert_chain_platform_specific(verifyCtx, utility::conversions::to_utf8string(host)))
-            {
-                return false;
-            }
-            else
-            {
-                return m_http_client->client_config().invoke_certificate_chain_callback(host, get_X509_cert_chain_encoded_data(verifyCtx));
-            }
+            auto chainFunc = [this](const std::shared_ptr<certificate_info>& cert_info) {
```

Comment:
That's what was happening before, i just changed it back to what we where doing before.
I think the lifecycle is good the executed in a synchronous call.